packages/neo-one-server-plugin-project/src/utils/loadProjectConfig.ts

```diff
@@ -26,3 +33,35 @@ export const loadProjectConfig = async (rootDir: string): Promise<ProjectConfig>
 
   return validateConfig(rootDir, result === null ? {} : result.config);
 };
+
+// tslint:disable-next-line: no-any
+const getPkgDependencies = (pkg: any): Set<string> => {
+  const dependencies = pkg.dependencies === undefined ? [] : Object.keys(pkg.dependencies);
+  const devDependencies = pkg.devDependencies === undefined ? [] : Object.keys(pkg.devDependencies);
+
+  return new Set(dependencies.concat(devDependencies));
+};
+
+const getProjectFramework = async (rootDir: string): Promise<CodegenFramework> => {
+  const pkgPath = path.resolve(rootDir, 'package.json');
+  const pkgExists = await fs.pathExists(pkgPath);
+  if (!pkgExists) {
+    return 'none' as const;
+  }
+  const pkg = await fs.readJSON(path.resolve(rootDir, 'package.json'));
+  const dependencies = getPkgDependencies(pkg);
+
+  if (dependencies.has('react') || dependencies.has('react-dom')) {
```

Comment:
Does this work with create-react-app? I don't recall if react is explicitly a dependency or just cra is.

Reply:
Yes when I did
```
npx create-react-app token --typescript
```
react and react-dom were explicit dependencies.